rest-server/src/middlewares/parameter.js

```diff
@@ -20,18 +20,19 @@
 const Joi = require('joi');
 const logger = require('../config/logger');
 
-
 /**
  * Validate parameters.
  */
 const validate = (schema) => {
   return (req, res, next) => {
     Joi.validate(req.body, schema, (err, value) => {
       if (err) {
-        logger.warn('parameters validation error\n%s', err.stack);
+        errorType = 'ParameterValidationError';
+        errorMessage = 'Could not validate request data.\n' + err.stack;
+        logger.warn('[%s] %s', errorType, errorMessage);
         return res.status(500).json({
-          error: 'ParameterValidationError',
-          message: 'could not validate request data'
+          error: errorType,
+          message: errorMessage
```

Comment:
please add trailing comma to pass lint check

Reply:
Will do.